halo2_gadgets/src/sinsemilla/primitives.rs

```diff
@@ -200,6 +201,21 @@ impl CommitDomain {
         }
     }
 
+    /// Constructs a new `CommitDomain` from `hash_personalization` a personalization for hashing
+    /// and `blind_personalization` another personalization for blinding.
+    pub fn new_with_two_personalizations(
```

Comment:
Some naming update might be needed:
https://github.com/QED-it/halo2/pull/14/commits/a68d519e0be3ac85d9b70faa5d1ca4290b91a0b4

We can use the current naming until Zcash team will review the PR.